keps/sig-auth/20190607-certificates-api.md

```diff
@@ -325,8 +377,13 @@ type CertificateSigningRequestStatus struct {
 }
 
 type CertificateSigningRequestCondition struct {
-  // request approval state, currently Approved or Denied.
+  // conditions, including ones that indicate request approval ("Approved", "Denied") and failure ("Failed").
+  // Approved/Denied conditions are mutually exclusive.
+  // Approved, Denied, and Failed conditions cannot be removed once added.
   Type RequestConditionType
+  // status of the condition. Valid values are True, False, and Unknown.
+  // Approved, Denied, and Failed conditions are limited to a value of True.
+  Status v1.ConditionStatus
```

Comment:
This is a new field so should have a default value and be optional right? From our previous discussions, I think this also needs to default to *True* in order to maintain backward compatibility with clients who already use their own condition types.

Reply:
> This is a new field so should have a default value and be optional right?

In v1beta1, yes. In v1, no.

> From our previous discussions, I think this also needs to default to True in order to maintain backward compatibility with clients who already use their own condition types.

In v1beta1, yes.